Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -73,19 +74,40 @@
             rnd=rnd,
             outputDirRoot=outputDir,
         )
+    if not g4_simulation:
+        addFatras(
+            s,
+            trackingGeometry,
+            field,
+            ParticleSelectorConfig(
+                eta=(-3.0, 3.0), pt=(150 * u.MeV, None), removeNeutral=True
+            )
+            if ttbar_pu200
+            else ParticleSelectorConfig(),
+            outputDirRoot=outputDir,
+            rnd=rnd,
+        )
+    if g4_simulation:
+        if Sequencer.config.numThreads != 1:
```

Comment:
```suggestion
        if s.config.numThreads != 1:
```

otherwise it would check on the default no?

Reply:
fixed